DocumentedExamples/Querying_Scalar_Quantities_and_Annually_Averaged_Timeseries.ipynb

```diff
@@ -0,0 +1,2091 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    for experiment in experiments:
</pre><p> </p><p>Loop over the dictionary you've just made:</p><p> </p><pre class="ql-syntax" spellcheck="false">for experiment, result in results.items():
  result.plot(label=experiment, ax=ax)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>